py3status/modules/arch_updates.py

```diff
@@ -117,6 +135,36 @@ def _get_pikaur_updates(self):
             return None if ce.error else 0
 
     def arch_updates(self):
+        cached = self._get_cached_value()
+        if cached is not None:
+            return {"full_text": cached}
+
+        full_text = self._get_display_text()
+
+        self.py3.storage_set(CACHE_KEY_TEXT, full_text)
+        self.py3.storage_set(CACHE_KEY_TIMESTAMP, time.time())
+
+        return {"full_text": full_text}
```

Comment:
I'm not using Arch right now, but this looks like `arch_updates` module may be constantly refreshed now... in order to avoid running updater command. Unconfirmed, but something to think about.

The better way just may be to document users to use this hook with `py3-cmd refresh arch_updates` and disable `cache_timeout` instead. https://archlinux.org/pacman/alpm-hooks.5.html